### DISCUSSION TITLE
Making Country selector but couldn't sort by Country's name.

### DISCUSSION THREAD

**Discussion by spencerchang** [annexare/Countries#130] [Q&A] [answered]
EX: South Africa coutryCode is ZA, will be sort on the list.
Sometimes user feels hard to find countries from the list.

Any suggestions?
Thank you.

**Comment by dmythro** [comment, marked as answer]
Use `getCountryDataList()` or `Object.entries()` approach to get all data as array of objects and sort by any property you need.
